backend/core/package.json

```diff
@@ -125,7 +125,8 @@
     "testEnvironment": "node"
   },
   "engines": {
-    "node": "12",
+    "node": "14",
```

Comment:
I think you'll want to revert this part since we don't want to upgrade our node engine.